packages/components/src/card/card-footer/README.md

```diff
@@ -39,40 +41,34 @@ const Example = () => (
 );
 ```
 
-Check out [the documentation](../../flex/flex/README.md) on `<Flex/>` for more details on layout composition.
+Check out [the documentation](../..//flex/flex/README.md) on `Flex` for more details on layout composition.
 
 ## Props
 
-### isBorderless
+Note: This component is connected to [`Card`'s Context](../card/README.md#context). Passing the `size` and `isBorderless` props directly to this component will override the value derived from context.
```

Comment:
> override the value derived from context

This is a little confusing still.... should we mention that it only derives these props if they are passed into a parent `Card` component?

Reply:
I tried rewriting it in https://github.com/WordPress/gutenberg/pull/32566/commits/4fe415f8a345350ca0da6cc9768d7abbc8fb936f, let me know if it's still not clear enough.